api/admin.go

```diff
@@ -160,8 +162,10 @@ func saveConfig(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	utils.SaveConfig(utils.CfgFileName, cfg)
 	utils.LoadConfig(utils.CfgFileName)
-	json := utils.Cfg.ToJson()
-	w.Write([]byte(json))
+
+	rdata := map[string]string{}
```

Comment:
Scroll through `api/admin.go` they're all over. Let's discuss in the dev meeting about what to do about them. I don't think we should have any APIs that don't have some sort of response.